go/obsclient/obsclient.go

```diff
@@ -80,3 +80,10 @@ func (oc *ObsClient) RollupHeaderByHash(hash gethcommon.Hash) (*common.Header, e
 	}
 	return rollupHeader, err
 }
+
+// Health returns the health of the node.
+func (oc *ObsClient) Health() (bool, error) {
```

Comment:
Should this be `IsHealthy`? Your call.

Reply:
I would like to be returning a health json object in the future, depending on how we see it used on datadog.